docs/updates/April_2024.md

```diff
@@ -0,0 +1,107 @@
+---
+sidebar_position: 6
+---
+
+# April 2024
+
+## tl;dr
+
+- Interviewing a number of folks for the front end devs to help maintain
+  momentum after pushing most of the current team to the contract side.
+- Contract work is the main priority and the resources and attention have been
+  focused to execute.
+- Planning to announce the raise once we are in Audits.
+- Built content calendar for social to help build organic socials growth.
+- Defined team Vision, Mission, and Values.
+
+## Development
+
+### Hiring
+
+We have been actively interviewing candidates, thank you to anyone who connected
+us with candidates or shared recruiting resources. I was able to delegate much
+of the hiring process over to the team, only directly participating in final
+interviews. While we max out our engineering resources to complete the contract
+code, new hires will onboard to the front end and allow us to maintain momentum
+in the short term and execrate our velocity once contract are in audits. 
+
+### Contracts
+
+Last month I mentioned that we hired a new contract engineer and are giving two
+front end engineers contract work. A lot of work went into leveling up our newer
+engineers to understand and contribute to our contracts and those efforts are
+starting to manifest in improvements to our velocity.
+
+To give a bit more context, we have three remaining items to wrap up:
+
+- Interest accrual
+- Cascading liquidation protections
+- Liquidations
+
+At the start of last month, one of these items was owned by an engineer, but now
```

Comment:
"one of these items was owned by an engineer" ?
maybe "one engineer"?

Reply:
reworded